### PR TITLE
EDSC-1000 CSS change to prevent text on map in detail view.

### PR DESCRIPTION
**Pull request by mightynimble**

### CODE REVIEW

File: app/assets/stylesheets/modules/media_queries.css.scss [hunk 1]
```diff
@@ -19,7 +19,7 @@
 	}
 }
 
-@media (max-width: 1349px) {
+@media (max-width: 1549px) {
```
Comment (bilts) [inline app/assets/stylesheets/modules/media_queries.css.scss]: This approach seems fragile and likely to break again. Can you find something more robust?

Maybe use flexbox with https://github.com/10up/flexibility ?

**Comment by bilts** [comment]
Needs polyfill for IE < 11 as mentioned in my comment.

**Comment by mightynimble** [comment]
Tested in IE9 and IE11. The page looks ok without the polyfill setting. With ```-js-display```, the spatial/temporal part hides behind the mini map when the width of the detail panel shrinks..

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -30,6 +30,7 @@
     <% end %>
     <%= yield :javascript %>
     <%= javascript_include_tag "jquery.simulate" if Rails.env.test? %>
+    <!--[if lt IE 10]> <%= javascript_include_tag "flexibility" %> <![endif]-->
```
Comment (bilts) [inline app/views/layouts/application.html.erb]: Does IE 10 work?

File: app/assets/stylesheets/modules/master_overlay.css.scss [hunk 3]
```diff
@@ -418,13 +418,14 @@ $parent-width: $parent-split * 0.21;
 }
 
 .row {
+  -js-display: flex;
```
Comment (bilts) [inline app/assets/stylesheets/modules/master_overlay.css.scss]: I thought you had said -js-display messed up IE 11. Is it only if flexibility is loaded?

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (mightynimble) [inline app/views/layouts/application.html.erb]: I don't have IE10 to test. But Flex is supported by IE10 by default. So I think it should behave like IE11.

File: app/assets/stylesheets/modules/master_overlay.css.scss (hunk 3 above)
Comment (mightynimble) [inline app/assets/stylesheets/modules/master_overlay.css.scss]: Besides -js-display, the fixed-width mini map also contributes to the issue in IE11. Therefore, I made changes to .flex-column class as well.

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (bilts) [inline app/views/layouts/application.html.erb]: I thought IE 10 required a vendor prefix, `-ms-`

File: app/assets/stylesheets/modules/master_overlay.css.scss (hunk 3 above)
Comment (bilts) [inline app/assets/stylesheets/modules/master_overlay.css.scss]: So, does this line still cause problems in IE 11?

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (mightynimble) [inline app/views/layouts/application.html.erb]: I am downloading an IE10 VM. I will test it when it finishes, with and without -ms- prefix.

File: app/assets/stylesheets/modules/master_overlay.css.scss (hunk 3 above)
Comment (mightynimble) [inline app/assets/stylesheets/modules/master_overlay.css.scss]: No. IE11 is fine now.

**Comment by mightynimble** [comment]
So, another commit. This fixes layouts on REAL IE boxes. ( Although it looks alright in VM IE9-11, the two-column detail page displays as a single column on physical Windows machines:) )

File: app/assets/stylesheets/modules/media_queries.css.scss (hunk 1 above)
Comment (bilts) [inline app/assets/stylesheets/modules/media_queries.css.scss]: Are these media changes still needed? Why?

Comment (mightynimble) [inline app/assets/stylesheets/modules/media_queries.css.scss]: I have no idea how they get in.. I will remove it.

**Pull request merged**